src/main/resources/xslt/preservica2solr.xsl

```diff
@@ -91,16 +164,17 @@
           </xsl:when>
         </xsl:choose>
       </xsl:for-each>
-
     </xsl:if>
 
+    <!-- resource description is not a description of what the resource contains.
+         It's rather a description of the type of resource. -->
     <f:string key="resource_description">
       <xsl:value-of select="pbcoreInstantiation/formatMediaType"/>
     </f:string>
 
-
+    <!-- Title extraction -->
     <xsl:for-each select="pbcoreTitle">
-      <xsl:if test="titleType = 'titel'">
+      <xsl:if test="titleType = 'titel' and title != ''">
```

Comment:
The need for a check for empty string is a bit worrying. Are there other properties that can be defined in the Preservica output that can have empty values?

Reply:
I am not sure about this, but my initial thought is that many fields could be generated without content. I believe the fields are created even though they are empty, so that curators can create the metadata in Kuana - I can point you to some internal documentation of this.

Reply:
No thank you :-)

This is too much nitpicking in a semi-random direction from my side.